static/strategies.config.php

```diff
@@ -29,4 +31,18 @@
 		Type\SyslogLogger::class => ['syslog'],
 		Type\StreamLogger::class => ['stream'],
 	],
+	Cache\Capability\ICanCache::class => [
+		Cache\Type\APCuCache::class      => ['apcu'],
+		Cache\Type\DatabaseCache::class  => ['database', ''],
```

Comment:
Please define a constant with the value `'default'` that you can refer here and anywhere else you've used empty string to mean the default strategy.

The location and the name of the constant are left to you, although a mention of "default" would be nice.